packages/vue/docs/releases/v0.9.0.md

```diff
@@ -1,15 +1,6 @@
-# Releases
+# version 0.9.0
```

Comment:
version in the tables on this page should also be changed from 0.9.1 to 0.9.0 :)

Reply:
fixed